tools/invoke-tests

```diff
@@ -58,6 +58,8 @@ while [[ $# -gt 0 ]]; do
     esac
 done
 [[ "$*" ]] && TESTS="$*"
+# Run all tests from t by default (instead of all .t files)
+[[ -z "$TESTS" ]] && TESTS=t
```

Comment:
Hm, the problem is, below in line 81 we have 
```
TESTS=${TESTS:-"-r t xt"}
```

Reply:
🤦🏼‍♀️